### DISCUSSION TITLE
Got error during creating a step for flow

### DISCUSSION THREAD

**Discussion by Omkartomar93** [composer/composer#11819] [Q&A / Support]
Hey ... i got this error on my console -   RingleSoft\LaravelProcessApproval\ProcessApproval::createStep(): Argument #1 ($flowId) must be of type int, string given, called in C:\php8\htdocs\Prog-HRM\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php on line 356

**Comment by Seldaek** [comment]
Sorry but this does not seem related to Composer at all.
